applications/nrf_desktop/Kconfig.defaults

```diff
@@ -22,12 +22,10 @@ choice LIBC_IMPLEMENTATION
 	  Use minimal libc implementation to reduce memory footprint.
 endchoice
 
-config LTO
-	default y
-	help
-	  nRF Desktop enables LTO to limit memory usage and improve performance.
-
-config ISR_TABLES_LOCAL_DECLARATION
+config DESKTOP_LTO_DEFAULT
```

Comment:
What does "DEAFULT" mean in name: DESKTOP_LTO_DEFAULT shouldn't DESKTOP_LTO be enough?

Reply:
Initially I named it `DESKTOP_LTO`, but then I realized that the option would stay enabled if user disables the implied options. That's why I decided to rename it.

Reply:
Maybe: DESKTOP_LTO_DEFAULTS
or DESKTOP_LTO_DEFAULT_SETTINGS

Reply:
Renamed to `DESKTOP_LTO_DEFAULTS`